website/src/pages/docs/guides/react-vue.mdx

```diff
@@ -480,14 +480,35 @@ From simple Queries to more advanced Fragments-based ones, GraphQL Code Generato
 
 **What's next?**
 
-To get the best GraphQL development experience, we recommend installing the [GraphQL VSCode extension](https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql) to get:
+To get the best GraphQL development experience, we recommend installing the [GraphQLSP](https://github.com/0no-co/GraphQLSP) package to get:
```

Comment:
Should we add a screenshot to demonstrate the functionality? It could help convince people (optional though).